libsigner/src/v0/messages.rs

```diff
@@ -851,14 +925,20 @@ mod test {
         let rejection = BlockRejection::new(
             Sha512Trunc256Sum([0u8; 32]),
             RejectCode::ValidationFailed(ValidateRejectCode::InvalidBlock),
+            &StacksPrivateKey::new(),
+            thread_rng().next_u32() % 2 == 0,
```

Comment:
How about this instead?

```suggestion
            thread_rng().gen_bool(0.5),
```

Also two more of these below